<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProfileDialog</class>
 <widget class="QDialog" name="ProfileDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>486</width>
    <height>543</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Profiles Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>481</width>
     <height>421</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab1">
    <attribute name="title">
     <string>Chart</string>
    </attribute>
    <widget class="QCustomPlot" name="customPlot" native="true">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>489</width>
       <height>301</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>CrossCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string extracomment="Temperature trace">Temperatur traker</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Params</string>
    </attribute>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>290</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QTabWidget" name="tabWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>481</width>
       <height>371</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_3">
      <property name="font">
       <font>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <attribute name="icon">
       <iconset resource="../resources.qrc">
        <normaloff>:/images/copy.png</normaloff>
        <normalon>:/images/write.png</normalon>:/images/copy.png</iconset>
      </attribute>
      <attribute name="title">
       <string>Profile</string>
      </attribute>
      <widget class="QWidget" name="formLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>461</width>
         <height>221</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="profile_formLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="profile_label_1">
          <property name="text">
           <string>Profiles</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QComboBox" name="profileComboBox">
          <property name="toolTip">
           <string>toolTip</string>
          </property>
          <property name="currentText">
           <string/>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="profile_label_2">
          <property name="text">
           <string>Name</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="pfofileNameEdit">
          <property name="toolTip">
           <string extracomment="Profile name comment">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Profile name&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="profile_label_3">
          <property name="text">
           <string>Description</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QTextEdit" name="profileDescription_textEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>60</height>
           </size>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
          <property name="toolTip">
           <string>profile description</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="profile_label_4">
          <property name="text">
           <string>File name</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QLineEdit" name="profileFile_Edit"/>
        </item>
        <item row="4" column="0">
         <widget class="QLabel" name="profile_label_6">
          <property name="text">
           <string>Default</string>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QCheckBox" name="profileDefault_checkBox">
          <property name="text">
           <string>default check</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QPushButton" name="profile_pushButton_Save">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>240</y>
         <width>61</width>
         <height>27</height>
        </rect>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>Temperature</string>
      </attribute>
      <widget class="QWidget" name="horizontalLayoutWidget_2">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>9</y>
         <width>461</width>
         <height>321</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Temperature</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="temprature_lineEdit">
            <property name="inputMask">
             <string>0000</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Proportional term Kp</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="tpropotional_lineEdit">
            <property name="maxLength">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Integral term Ki</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="tintegral_lineEdit">
            <property name="inputMask">
             <string>000.000</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Derivative term Kd</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="tderivative_lineEdit">
            <property name="inputMask">
             <string>000.0000</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_4">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="hOMING_DIR_MASKLabel">
            <property name="text">
             <string>HOMING_DIR_MASK</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="hOMING_DIR_MASKLineEdit">
            <property name="toolTip">
             <string>HOMING_DIR_MASK</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="hOMING_FEED_RATELabel">
            <property name="text">
             <string>HOMING_FEED_RATE</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="hOMING_FEED_RATELineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="hOMING_SEEK_RATELabel">
            <property name="text">
             <string>HOMING_SEEK_RATE</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="hOMING_SEEK_RATELineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="hOMING_D_DELAYLabel">
            <property name="text">
             <string>HOMING_D_DELAY</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="hOMING_D_DELAYLineEdit">
            <property name="toolTip">
             <string>HOMING_DEBOUNCE_DELAY</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="hOMING_PULLOFFLabel">
            <property name="text">
             <string>HOMING_PULLOFF</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="hOMING_PULLOFFLineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_5">
      <attribute name="title">
       <string>Axis</string>
      </attribute>
      <widget class="QWidget" name="horizontalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>471</width>
         <height>341</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>10</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>10</number>
        </property>
        <property name="bottomMargin">
         <number>10</number>
        </property>
        <item>
         <layout class="QFormLayout" name="formLayout_2">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>X_STEPS_PER_MM</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Y_STEPS_PER_MM</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="lineEdit_2"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Z_STEPS_PER_MM</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEdit_3"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>X_MAX_RATE</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEdit_4"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Y_MAX_RATE</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEdit_5"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Z_MAX_RATE</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="lineEdit_6"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>X_ACCELERATION</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Y_ACCELERATION</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEdit_7"/>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lineEdit_8"/>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Z_ACCELERATION</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QLineEdit" name="lineEdit_9"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QFormLayout" name="formLayout_3">
          <property name="leftMargin">
           <number>10</number>
          </property>
          <property name="topMargin">
           <number>10</number>
          </property>
          <property name="rightMargin">
           <number>10</number>
          </property>
          <property name="bottomMargin">
           <number>10</number>
          </property>
          <item row="0" column="0">
           <widget class="QLabel" name="x_MAX_TRAVELLabel">
            <property name="text">
             <string>X_MAX_TRAVEL</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="x_MAX_TRAVELLineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="y_MAX_TRAVELLabel">
            <property name="text">
             <string>Y_MAX_TRAVEL</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="y_MAX_TRAVELLineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="z_MAX_TRAVELLabel">
            <property name="text">
             <string>Z_MAX_TRAVEL</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="z_MAX_TRAVELLineEdit"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="sTEP_PULSE_MICROSECONDSLabel">
            <property name="toolTip">
             <string>STEP_PULSE_MICROSECONDS</string>
            </property>
            <property name="text">
             <string>STEP_PULSE_MS</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="step_PULSE_MICROSECONDSLineEdit"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="s_iNVERT_MASKLabel">
            <property name="toolTip">
             <string>STEPPING_INVERT_MASK</string>
            </property>
            <property name="text">
             <string>S_invert_MASK</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="invert_MASKLineEdit"/>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="d_INVERT_MASKLabel">
            <property name="toolTip">
             <string>DIRECTION_INVERT_MASK</string>
            </property>
            <property name="text">
             <string>D_invert_MASK</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="d_INVERT_MASKLineEdit"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="idle_LOCK_TIMELabel">
            <property name="toolTip">
             <string>STEPPER_IDLE_LOCK_TIME</string>
            </property>
            <property name="text">
             <string>idle_LOCK_TIME</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="idle_LOCK_TIMELineEdit"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="aRC_TOLERANCELabel">
            <property name="text">
             <string>ARC_TOLERANCE</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="aRC_TOLERANCELineEdit"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Status</string>
    </attribute>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>290</y>
       <width>85</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Refresh</string>
     </property>
    </widget>
    <widget class="QListView" name="listView">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>1</y>
       <width>471</width>
       <height>261</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>510</y>
     <width>451</width>
     <height>20</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
